packages/shared/src/hooks/useInputFieldFunctions.ts

```diff
@@ -37,58 +36,48 @@ function useInputFieldFunctions<
     focusInput,
     setInput,
   } = useInputField<T>(value, valueChanged);
+
   const [inputLength, setInputLength] = useState<number>(undefined);
   const [validInput, setValidInput] = useState<boolean>(undefined);
-  const [idleTimeout, clearIdleTimeout] = useDebounceFn(() => {
```

Comment:
It is important that we keep the debounce, so that we don't have to fire the same function over and over again when we can do it when we notice the user stopped typing.